### PR TITLE
New package: pyradio-0.7.9

### PR DESCRIPTION
**Pull request by lugubris**
fixes https://github.com/void-linux/void-packages/issues/14181

### CODE REVIEW

**Comment by lugubris** [comment]
I need some help with this template. There are 3 ways install can proceed -> install mpv & socat, install mplayer or install vlc

can someone point me to a similar template? xbps should ask user which backend it should use.

**Comment by hfel** [comment]
Or in case xbps cannot prompt the user, just let the user know that they have to have one of the three. I believe that's how it works with LightDM and its various greeters.

**Comment by lugubris** [comment]
Tõnis Rand @ 2019-09-04 15:00 UTC:

> Or in case xbps cannot prompt the user, just let the user know that they have to have one of the three. I believe that's how it works with LightDM and its various greeters.

I added a install message, same as how lightdm does it.
--
                                Lugubris

**Comment by lugubris** [comment]
> => ERROR: pyradio-0.7.9_1: do_build: 'python3 setup.py build ${make_build_args}' exited with 1
> 
> => ERROR:   in do_build() at common/build-style/python3-module.sh:16
> 
> The command "if [ -z "$XLINT" ]; then docker exec -t void hostrepo/common/travis/build.sh
> $BOOTSTRAP $ARCH; fi" exited with 1.

build is failing, how do I fix it?

pyradio can use both python3 and python2 is there an option to ask user which one to use?

**Comment by zdtcd** [comment]
@lugubris It seems like this is an executable package.
We should just go with python3, since python2 will be EOL soon.

File: srcpkgs/pyradio/template [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Template file for 'pyradio'
+pkgname=pyradio
+version=0.7.9
+revision=1
+build_style=python-module
```
Comment (zdtcd) [inline srcpkgs/pyradio/template]: s/python/python3

File: srcpkgs/pyradio/template [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# Template file for 'pyradio'
+pkgname=pyradio
+version=0.7.9
+revision=1
```
Comment (zdtcd) [inline srcpkgs/pyradio/template]: add this line:
```
arch=noarch
```

File: srcpkgs/pyradio/template [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+build_style=python-module
+pycompile_module="pyradio"
+hostmakedepends="python3"
```
Comment (zdtcd) [inline srcpkgs/pyradio/template]: s/python3/&-setuptools

File: srcpkgs/pyradio/template [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+maintainer="Lugubris <lugubris@disroot.org>"
+license="MIT"
+homepage="http://www.coderholic.com/pyradio/"
+changelog="https://raw.githubusercontent.com/coderholic/pyradio/master/ChangeLog"
+distfiles="https://github.com/coderholic/pyradio/archive/${version}.tar.gz"
+checksum=a11e487a715c413cff5f80967dec41ba38888a1b1901237bb1fa018c8ac49499
+
+post_install() {
+	vman pyradio.1
+	vlicense licenses/mit.txt LICENSE
```
Comment (zdtcd) [inline srcpkgs/pyradio/template]: `vlicense LICENCE`

Note the second capitalised C

**Comment by lugubris** [comment]
Danh Doan @ 2019-09-05 15:32 UTC:

> zdtcd requested changes on this pull request.
>
> Note the second capitalised C

@zdtcd Interesting [0]

[0] -> https://www.gcse.com/english/licence.htm

--
                                Lugubris

**Comment by lugubris** [comment]
build passed

**Comment by zdtcd** [comment]
On September 5, 2019 4:04:48 PM UTC, Lugubris <notifications@github.com> wrote:
>
>Danh Doan @ 2019-09-05 15:32 UTC:
>
>> zdtcd requested changes on this pull request.
>>
>> Note the second capitalised C
>
>@zdtcd Interesting [0]
>
>[0] -> https://www.gcse.com/english/licence.htm
>
>--
>                                Lugubris

Just `vlicense LICENCE` is enough.
Reason: the licence filename is LICENCE
-- 
Danh

**Comment by lugubris** [comment]
Danh Doan @ 2019-09-06 12:21 UTC:

> On September 5, 2019 4:04:48 PM UTC, Lugubris <notifications@github.com> wrote:
>>
>>Danh Doan @ 2019-09-05 15:32 UTC:
>>
>>> zdtcd requested changes on this pull request.
>>>
>>> Note the second capitalised C
>>
>>@zdtcd Interesting [0]
>>
>>[0] -> https://www.gcse.com/english/licence.htm
>
> Just `vlicense LICENCE` is enough.
> Reason: the licence filename is LICENCE

resolved

--
                                Lugubris

**Pull request merged**